guide/en/start/databases.md

```diff
@@ -3,13 +3,28 @@
 Yii doesn't dictate using a particular database or storage for your application.
 There are many ways you can work with relational databases:
 
-- [Yii Database](https://github.com/yiisoft/db)
-- [Doctrine](https://www.doctrine-project.org/) via [Yii Doctrine package](https://github.com/stargazer-team/yii-doctrine)
+- [DB](https://github.com/yiisoft/db)
+- [Active Record](https://github.com/yiisoft/active-record)
 - [Cycle](https://github.com/cycle) via [Yii Cycle package](https://github.com/yiisoft/yii-cycle)
+- [Doctrine](https://www.doctrine-project.org/) via [Yii Doctrine package](https://github.com/stargazer-team/yii-doctrine)
 - [PDO](https://www.php.net/manual/en/book.pdo.php)
 
 For non-relational ones, there are usually official libraries available:
 
 - [ElasticSearch](https://github.com/elastic/elasticsearch-php)
 - [Redis](https://redis.io/docs/clients/#php)
 - ...
+
+In this guide, we will focus on working with relational databases using DB.
+
+## Configuring connection
+
+## Creating and applying migration
+
+## Inserting
+
+## Selecting
+
+## Using data package
+
+### Pagination
```

Comment:
Is this part will be copied from the doc of the relevant packages?

Reply:
Partially. Here we need to show how to configure it in DI container of the app's configs.